packages/flutter/lib/src/rendering/platform_view.dart

```diff
@@ -94,13 +94,13 @@ class RenderAndroidView extends RenderBox {
 
   /// Which gestures should be forwarded to the Android view.
   ///
-  /// The gesture recognizers on this list participate in the gesture arena for each pointer
-  /// that was put down on the render box. If any of the recognizers on this list wins the
+  /// Gesture recognizers created by factories in this set participate in the gesture arena for each
+  /// pointer that was put down on the render box. If any of the recognizers on this list wins the
   /// gesture arena, the entire pointer event sequence starting from the pointer down event
   /// will be dispatched to the Android view.
-  set gestureRecognizers(List<OneSequenceGestureRecognizer> recognizers) {
+  set gestureRecognizers(Set<Factory<OneSequenceGestureRecognizer>> recognizers) {
```

Comment:
I'd probably use `value` as the name of the parameter here, because it's confusing that "recognizers" is used to set "_gestureRecognizers".

Reply:
(as in, it's not immediately clear when reading the code which is the field and which is the incoming parameter)